Frameworks/CoreText/CTLine.mm

```diff
@@ -65,6 +66,27 @@ CTLineRef CTLineCreateWithAttributedString(CFAttributedStringRef string) {
     return string ? static_cast<CTLineRef>(_DWriteGetLine(string)) : nil;
 }
 
+CTLineRef CTLineCreateWithAttributedStringAndWidth(CFAttributedStringRef string, CFRange range, double width) {
```

Comment:
>CTLineCreateWithAttributedStringAndWidth [](start = 10, length = 40)

please follow naming convention to identify this as an internal method.
Same below.